eclass/kernel-install.eclass

```diff
@@ -569,40 +578,39 @@ kernel-install_src_test() {
 kernel-install_pkg_preinst() {
 	debug-print-function ${FUNCNAME} "${@}"
 
-	local dir_ver=${PV}${KV_LOCALVERSION}
-	local kernel_dir=${ED}/usr/src/linux-${dir_ver}
-	local relfile=${kernel_dir}/include/config/kernel.release
+	# Set KV_FULL to ${PV}${KV_LOCALVERSION} if it hasn't
+	# been set elsewhere for backward compat
```

Comment:
`for backward compat` -> `for backward compat with older gentoo-kernel-bin` 

Lets explicitly say what we are being backwards compatible for, otherwise we'll be confused later.

Though I am a bit confused. This mismatch only occurs if there is _ in PV. Doesn't that mean that this mismatch is not present in the existing gentoo-kernel-bin's, and that therefore this backwards compatibility does nothing?

Reply:
By backward-compat, here I mean the fact that they do not set `KV_FULL` themselves, and do not inherit `kernel-build.eclass` to do it for them. Thus, the first time we reference `KV_FULL` inside `kernel-install.eclass`, we need to set it to _something_ so that `gentoo-kernel-bin`s still install properly. Setting it to what it's currently expected to be seems to be the sanest thing to do here. Perhaps we can look at setting KV_FULL in `gentoo-kernel-bin` going forward to make this unnecessary in the future?